docs/user/dql/functions.rst

```diff
@@ -1196,6 +1196,34 @@ Example::
     +---------------------+
 
 
+FROM_UNIXTIME
+-------------
+
+Description
+>>>>>>>>>>>
+
+Usage: Returns a representation of the argument given as a datetime or character string value. Perform reverse conversion for `UNIX_TIMESTAMP`_ function.
```

Comment:
Worth mentioning here how invalid inputs are handled and what inputs are invalid, including the maximum value.

Reply:
Thank you, updated in dfd2c1ea.